src/main/java/tools/jackson/databind/deser/impl/MethodProperty.java

```diff
@@ -22,23 +24,26 @@ public final class MethodProperty
 {
```

Comment:
Since this now handles Setters and Fields too, class Javadoc needs to be changed?